code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1140,7 +1140,7 @@
 /datum/species/proc/spec_hitby(atom/movable/AM, mob/living/carbon/human/H)
 	return
 
-/datum/species/proc/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, datum/martial_art/attacker_style = M.martial_art)
+/datum/species/proc/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, datum/martial_art/attacker_style = M.mind.martial_art)
```

Comment:
Mind is not guaranteed to exist here. This should default to some global/mob static and be passed explicitly in the call.